src/screens/SignUpScreen/index.tsx

```diff
@@ -13,17 +13,19 @@ import {
 const SignUpScreen: FC = () => {
   return (
     <KeyboardAvoidingView>
-      <SafeAreaView>
-        <StatusBar />
+      <TouchableWithoutFeedback onPress={Keyboard.dismiss}>
```

Comment:
@BamMironov @anq83 
I suggest we create a Container component, which will include StatusBar, SafeAreaView, and TouchableWithoutFeedback, KeyboardAvoidingView, etc(all configurable with boolean values if needed (KeyboardAvoidingView for example can be true or false)), also we can put global App margins, backgrounds, etc.